### PR TITLE
Mount SECRET_KEY into dev env & document it

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY
As part of https://github.com/ansible/awx/pull/9289, we need to actually mount the SECRET_KEY file into the awx container to be able to use the migrated data.  

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - Installer

##### AWX VERSION

```
devel
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by shanemcd — CHANGES_REQUESTED** [review]
In the migration steps you have:

```
$ ansible-playbook migrate.yml -e "migrate_local_docker=true" -e "postgres_data_dir=~/.awx/pgdocker" -e "old_docker_compose_dir=~/.awx/awxcompose"
```

This does not point to the inventory file (which includes the secret_key).

We need something like:

```
$ ansible-playbook -i tools/docker-compose/inventory tools/docker-compose/ansible/migrate.yml \
    -e "migrate_local_docker=true" \
    -e "postgres_data_dir=~/.awx/pgdocker" \
    -e "old_docker_compose_dir=~/.awx/awxcompose"
```

**Review by shanemcd — APPROVED** [review]

**Comment by rooftopcellist** [comment]
regate

**Review by ryanpetrello — APPROVED** [review]

**Comment by ryanpetrello** [comment]
recheck

File: awx/settings/defaults.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def IS_TESTING(argv=None):
 SITE_ID = 1
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = 'p7z7g1ql4%6+(6nlebb6hdk7sd^&fnjpal308%n%+p^_e6vo1y'
+SECRET_KEY = open('/etc/tower/SECRET_KEY', 'rb').read().strip()
```
Comment (ryanpetrello) [inline awx/settings/defaults.py]: Makes tests fail.  Should we put this line into `settings/development.py` instead of `defaults.py`?  cc @shanemcd

Comment (ryanpetrello) [reply awx/settings/defaults.py]: We could also wrap this in a `try...except...` and just generate a random value in the except block (so it'll just make a random one for tests)

Comment (rooftopcellist) [reply awx/settings/defaults.py]: The SECRET_KEY isn't getting mounted into the container at `/etc/tower` anymore for some reason, even though it is being rendered in the docker-compose.yml.

Comment (rooftopcellist) [reply awx/settings/defaults.py]: I was able to reproduce this locally, it looks like what is happening is that an old tools_awx_1 container was still around and was being used by the docker-compose, so the new volume-mount wasn't being created.  

After removing all containers and re-running `make docker-compose`, the secret_key was properly mounted into the container.

Comment (rooftopcellist) [reply awx/settings/defaults.py]: I could probably get tests passing by adding a `--force-recreate` flag in the Makefile.

Comment (ryanpetrello) [reply awx/settings/defaults.py]: Okay, that works too if we can get tests passing.

Comment (shanemcd) [reply awx/settings/defaults.py]: I'll take another look at this. I _really_ don't think we should have a default value for the secret key!

Comment (shanemcd) [reply awx/settings/defaults.py]: @rooftopcellist I don't think `--force-recreate` is going to help here. Can you back out that change please?

**Review by ryanpetrello — APPROVED** [review]

**Pull request merged**